Jenkinsfile

```diff
@@ -799,6 +799,7 @@ pipeline {
                         }
                         */
                         always {
+                            sh '''mv test_results/*.xml run_test.sh/'''
```

Comment:
If you move the `test_results/*.xml` files into  `run_test.sh/`, the `junit 'test_results/*.xml'` below will not be able to find them as they won't be in the right place, yes?

Reply:
Now, I am lost. I didn't have this move on my previous testing. Added them on the last few runs. Without move also, I don't see the XML files.